code/utils/string_utils.cpp

```diff
@@ -12,4 +12,11 @@ std::vector<std::string> split_string(const std::string& s, char delim) {
 	return elems;
 }
 
+bool isStringOneOf(const std::string& value, const std::vector<std::string>& candidates)
+{
+	return std::any_of(candidates.begin(), candidates.end(), [&value](const std::string& candidate) {
+		return !lcase_equal(value, candidate);
```

Comment:
Going over this in pieces, but at this part it looks like this will check if any string is not equal, which means it will almost always return true. So, the `lcase_equal` should not be negated, I reckon :)

Reply:
Leftover from when I changed from stricmp I bet. Fixed